### DISCUSSION TITLE
pop error window

### DISCUSSION THREAD

**Discussion by thijsdeschildre** [winder/Universal-G-Code-Sender#2035] [Help]
I'm trying to set up a machine with UGS. I'm trying to figure out some gcode commands. Every time the command is wrong, the program throws an error popup window which goes over the terminal, and I  can't read the most recent output.
Clicking the window doesn't close it, and makes it stay longer.
This is horribly annoying behaviour and I can't find a way to turn it off.

**Comment by breiler** [comment]
A picture (or a screenshot) is worth more than a thousand words. Because I don't understand what you mean.

What OS are you running? Which version of UGS are you using? Can you post log files?
